templates/docs/charger/keba-modbus_0.yaml

```diff
@@ -0,0 +1,17 @@
+product:
+  brand: KEBA
+  description: KeContact P20, P30, C/X Series
+capabilities: ["mA", "rfid"]
+requirements: ["sponsorship"]
+description: |
+  Erfordert Firmwareversion 3.10.16 (C-series) bzw. 1.11 (X-series)
```

Comment:
for c-series i would guess you have to have 3.10.42, otherwise modbus register 5050 is not available